### PR TITLE
Change ternary to tertiary for plugins

### PR DESCRIPTION
**Pull request by IntelOrca**

### CODE REVIEW

**Review by Gymnasiast — APPROVED** [review]
LGTM, but I don’t know much about plug-ins. Would thus appreciate an approval by another team member who knows more about them.

File: src/openrct2-ui/scripting/ScGraphicsContext.hpp [hunk 1]
```diff
@@ -42,7 +42,9 @@ namespace OpenRCT2::Scripting
             dukglue_register_property(
                 ctx, &ScGraphicsContext::secondaryColour_get, &ScGraphicsContext::secondaryColour_set, "secondaryColour");
             dukglue_register_property(
-                ctx, &ScGraphicsContext::ternaryColour_get, &ScGraphicsContext::ternaryColour_set, "ternaryColour");
+                ctx, &ScGraphicsContext::tertiaryColour_get, &ScGraphicsContext::tertiaryColour_set, "ternaryColour");
```
Comment (Broxzier) [inline src/openrct2-ui/scripting/ScGraphicsContext.hpp:45]: So this keeps support for `ternaryColour` around, shouldn't we mark this as deprecated, to remove it later?

**Review by Broxzier — COMMENTED** [review]
Not necessary, but for completeness sake the plug-in API version should be incremented.

File: src/openrct2-ui/scripting/ScGraphicsContext.hpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2-ui/scripting/ScGraphicsContext.hpp:45]: FWIW, I have created https://github.com/OpenRCT2/OpenRCT2/issues/16834 .

**Pull request merged**